file-formats/uist/type/zif_aff_uist_v1.intf.abap

```diff
@@ -0,0 +1,56 @@
+INTERFACE zif_aff_uist_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Properties</p>
+    "! Properties
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Title</p>
+      "! Space Template Title
+      "! $required
+      title         TYPE zif_aff_types_v1=>ty_description_100,
+      "! <p class="shorttext">Sort Priority</p>
+      "! Sort priority
+      "! $minimum: -999.999
+      "! $maximum: 999.999
+      sort_priority TYPE p LENGTH 7 DECIMALS 3,
+      "! <p class="shorttext">Base Space Template Name</p>
+      "! Base Space Template Name
```

Comment:
I'm not sure about SAP Terms here, but I think at least 'name' should be lower case, no?
```suggestion
      "! Base Space Template name
```

Reply:
maybe we could change it to "Name of the Base Space Template"? also not sure about the writing of "Base Space Template"

Reply:
We change it to "Name of the base space template" - as description .
... Write the description in sentence case .... - from documentation.